cloud/linode/vpc.go

```diff
@@ -55,3 +57,22 @@ func GetVPCID(client client.Client, vpcName string) (int, error) {
 	}
 	return 0, vpcLookupError{vpcName}
 }
+
+// GetVPCIPAddresses returns vpc ip's for given VPC label
+func GetVPCIPAddresses(ctx context.Context, client client.Client, vpcName string) ([]linodego.VPCIP, error) {
+	vpcID, err := GetVPCID(ctx, client, strings.TrimSpace(vpcName))
+	if err != nil {
+		return nil, err
+	}
+	resp, err := client.ListVPCIPAddresses(ctx, vpcID, linodego.NewListOptions(0, ""))
+	if err != nil {
+		if strings.Contains(err.Error(), "Not found") {
```

Comment:
does the linode API return a 404 for this or just a 400? If the latter, I'm fine with leaving this as is.

Reply:
It returns a 404 with `{"errors": [{"reason": "Not found"}]}`

Reply:
Ah okay, in that case, can we switch to doing `if linodego.ErrHasStatus(err, 404) {` instead? I think checking the error message makes more sense in the case of 400s since they're pretty generic.